private/buf/bufgen/testdata/v1/go_gen_error1.yaml

```diff
@@ -1,7 +1,12 @@
 version: v1
```

Comment:
I renamed `go_gen_error1.yaml` to `gen_success_7.yaml`, but it left the error files missing `go_gen_error1.yaml`., which is why I renamed `go_gen_error7.yaml` to `go_gen_error1.yaml` to fill the gap. (This is ok because there was not a strong logical order among the test files)